.github/workflows/ci.yml

```diff
@@ -15,6 +15,7 @@ on:
     tags: [v*]
 
 env:
+  JAVA_TOOL_OPTIONS: '-XX:+UseG1GC -XX:MaxHeapFreeRatio=20 -XX:MaxRAMPercentage=95.0 -XX:MinHeapFreeRatio=10 -XX:+UseStringDeduplication'
```

Comment:
`-XX:MaxRAMPercentage=95.0` does this leave enough room for forked processes, such as the Node.js and Scala Native test runners?

Reply:
Probably not...but it would only allocate up to 95% if the JVM actually needed it too. So if we got into such a situation, something would fail no matter what. I usually set it to 90-95 in containers, as opposed to on my development machine where I don't want some JVM process creating a bunch of swap.